source/agora/stats/EndpointReqestStats.d

```diff
@@ -0,0 +1,48 @@
+module agora.stats.EndpointReqestStats;
```

Comment:
I am going to use some compile time machinery to reuse this code accross different statistics, so functions like increase_total_calls_by can be reused